### ISSUE TITLE
Suggest adding Default address column

### ISSUE THREAD

**Issue by hosiawak** [nvk/wallets-recovery#37]
In order to recover the user needs to try a few address types (if unknown).
It would be great to have all this info already available.
For starters we can use [this matrix](https://bitcoinops.org/en/compatibility/)

I'll create a PR if @nvk OKs it.

**Comment by nvk** [comment]
Many of these wallets will have more than one default no?

**Comment by nvk** [comment]
Maybe we should create another page called  WALLET-DETAILS.MD and add an extra for each there. They would be longer and deep linked into each wallet section. Alternatively a doc file for each `wallets/`WALLETNAME`.MD`

**Comment by hosiawak** [comment]
By default I mean whatever address type you get when you hit "Receive". It's usually just one type as seen on the above compatibility matrix. So for Trezor it's P2SH-P2WPKH but for Samourai P2WPKH  and so on.  Whatever type the user needs to select to have the greatest chance of recovery in Electrum (or similar software)

**Comment by hosiawak** [comment]
I agree yet another column will make this table a bit less readable and packed though so maybe a details page for each wallet is a better option. I'll create a PR for another column and then you can decide.

**Comment by nvk** [comment]
Uncertain how helpful that would be as wallets change their defaults as they progress. Maybe in this extended page/file we could have "Expected default addresses for this wallet:"
